### DISCUSSION TITLE
Watermeter: Alignment markers

### DISCUSSION THREAD

**Discussion by Fagott01** [jomjol/AI-on-the-edge-device#3239] [General]
### The Feature

Servus! Ich bin kein Informatiker und einfacher Nutzer. Ich versuche seit längeren Zeit das Projekt "Watermeter" zu realisieren und quelle mich langsam durch. Nachdem ewig lang die Abnahme eines Referenzbildes nicht funktionierte (irgendwann habe ich es doch hingekriegt, weiß auch nicht wie...), komme ich bei den "Alignment markers" nicht weiter. Es will einfach diese nicht speichern. Ich habe es von verschiedenen Browsers ausprobiert - das Ergebnis ist immer gleich. Dabei besteht die WLAN-Verbindung, auf dem Livestream ist alles gestochen scharf zu sehen. Woran kann es liegen, dass es so kompliziert ist??? Danke.

**Comment by SybexX** [comment]
Welche Version benutzt du?
Du könntest die aktuelle rolling Version ausprobieren, die einige der in 15.7.0 bestehenden Probleme behebt.
https://github.com/jomjol/AI-on-the-edge-device/actions/runs/10690676061

**Comment by Fagott01** [comment]
die habe ich schon drauf. Alles, was ich beschrieben habe, bezieht sich suf die aktuelle Version. Gesendet mit der GMX iPad AppAm 14.09.24 um 13:40 schrieb michael
                
            
            
                
                    Von: "michael" ***@***.***>Datum: 14. September 2024An: "jomjol/AI-on-the-edge-device" ***@***.***>Cc: "Author" ***@***.***>,"Fagott01" ***@***.***>Betreff: Re: [jomjol/AI-on-the-edge-device] Watermeter: Alignment markers (Issue #3235)
                
                
Welche Version benutzt du?
Du könntest die aktuelle rolling Version ausprobieren, die einige der in 15.7.0 bestehenden Probleme behebt.
https://github.com/jomjol/AI-on-the-edge-device/actions/runs/10690676061

**Comment by SybexX** [comment]
Ich habe es gerade nochmal getestet und es funktioniert ohne Probleme.
Schritt 1: Bereich mit der Maus auswählen
Schritt 2: Auf "Update Marker" klicken
Schritt 3: Auf "Save New Marker" klicken
Schritt 4: Reboot durchführen

**Comment by Fagott01** [comment]
die Reihenfolge ist klar. Ein Bereich lässt sich markieren Wenn ich auf Update Marker klicke, passiert nichts. Es soll jedoch ein Image gespeichert und angezeigt, oder? Alles weitere bringt mich nicht weiter, weil Referenzimage nicht abgelegt werden. Gesendet mit der GMX iPad AppAm 14.09.24 um 14:18 schrieb michael
                
            
            
                
                    Von: "michael" ***@***.***>Datum: 14. September 2024An: "jomjol/AI-on-the-edge-device" ***@***.***>Cc: "Author" ***@***.***>,"Fagott01" ***@***.***>Betreff: Re: [jomjol/AI-on-the-edge-device] Watermeter: Alignment markers (Issue #3235)
                
                
Ich habe es gerade nochmal getestet und es funktioniert ohne Probleme.
Schritt 1: Bereich mit der Maus auswählen
Schritt 2: Auf "Update Marker" klicken
Schritt 3: Auf "Save New Marker" klicken
Schritt 4: Reboot durchführen

**Comment by SybexX** [comment]
ja, so sollte es ablaufen.
Kannst du die Dateien von deiner SD(auser der wlan.ini) als zip hier hochladen, um zu gucken was da schief läuft?

**Comment by Fagott01** [comment]
Ok. mache ich im Verlauf des Tages. Ich bin aktuell nicht daheim. Aber vorab schon besten Dank für die Hilfe. Kann es evtl. mit Mac zusammen hängen?Gesendet mit der GMX iPad AppAm 14.09.24 um 14:30 schrieb michael
                
            
            
                
                    Von: "michael" ***@***.***>Datum: 14. September 2024An: "jomjol/AI-on-the-edge-device" ***@***.***>Cc: "Author" ***@***.***>,"Fagott01" ***@***.***>Betreff: Re: [jomjol/AI-on-the-edge-device] Watermeter: Alignment markers (Issue #3235)
                
                
ja, so sollte es ablaufen.
Kannst du die Dateien von deiner SD(auser der wlan.ini) als zip hier hochladen, um zu gucken was da schief läuft?

**Comment by SybexX** [comment]
Ich vermute, dass die Datei edit_alignment.html noch von der alten Version ist, da es beim Update ein Problem gab.

**Comment by Fagott01** [comment]
Ich schicke die Daten ind hoffe, dass du Recht hast. Gesendet mit der GMX iPad AppAm 14.09.24 um 14:34 schrieb michael
                
            
            
                
                    Von: "michael" ***@***.***>Datum: 14. September 2024An: "jomjol/AI-on-the-edge-device" ***@***.***>Cc: "Fagott01" ***@***.***>,"Author" ***@***.***>Betreff: Re: [jomjol/AI-on-the-edge-device] Watermeter: Alignment markers (Issue #3235)
                
                
Ich vermute, dass die Datei edit_alignment.html noch von der alten Version ist, da es beim Update ein Problem gab.

**Comment by SybexX** [comment]
Hin und wieder gibt es Probleme beim Update, entweder werden einige Dateien nicht kopiert oder werden beim Kopieren beschädigt. Daher ist es am sichersten, die Dateien manuell mit dem PC auf die SD zu kopieren.

**Comment by Fagott01** [comment]
… wie geschrieben, hatte ich von Anfang an Probleme damit. Zuerst konnte ich kein Referenzbild machen. Zwar zeigte es mir den Wasserzahler an, nach dem man den Button“Referenzbild speichern“ anklickte. gab es nur eine schwarze Fläche… Nach dem Update auf die aktuelle Version hat sich der Zustand nicht geändert. Ich bin gespannt, was für Feedback ich von dir bekomme. Fie Daten kommen im Verlauf des Tages. Vg Alex. Gesendet mit der GMX iPad AppAm 14.09.24 um 14:42 schrieb michael
                
            
            
                
                    Von: "michael" ***@***.***>Datum: 14. September 2024An: "jomjol/AI-on-the-edge-device" ***@***.***>Cc: "Author" ***@***.***>,"Fagott01" ***@***.***>Betreff: Re: [jomjol/AI-on-the-edge-device] Watermeter: Alignment markers (Issue #3235)
                
                
Hin und wieder gibt es Probleme beim Update, entweder werden einige Dateien nicht kopiert oder werden beim Kopieren beschädigt. Daher ist es am sichersten, die Dateien manuell mit dem PC auf die SD zu kopieren.

**Comment by Fagott01** [comment]
[Archiv.zip](https://github.com/user-attachments/files/17005007/Archiv.zip)
Hier sind die Daten

**Comment by Fagott01** [comment]
Ich habe immer noch kein Feedback erhalten! Nach dem ich verschiedene SD-Karten (marken und markenlose) sowie verschiedene Kapazitäten getestet habe, komme ich durch die Installationsroutine durch. Allerdings kann ich weiterhin keine Alignmebnt-Marker setzen. Diese werden nicht gespeichert. Alles andere lässt sich irgendwie anzeigen und speichern. Neu ist, dass ich eine Fehlermeldung bekomme: "The Version of the web interface (...) does not Match the firmware Version (...)! It is suggested to keep them on the same Version" Wie bekomme ich das hon? Das Löschen des Browser-Caches hat nichts gebracht.

**Reply by SybexX** [reply]
Diese Fehlermeldung kommt, wenn die Dateien auf der SD nicht mit der Firmware Version übereinstimmen.
Wenn du die passenden SD-Dateien nicht verwendest, kann es zu Problemen kommen.
Hast du die Dateien nachträglich bearbeitet oder eins zu eins von der SD kopiert?
Den mit der config.ini, die dabei ist, kann der ESP garnicht richtig starten, da sehr viele Einträge fehlen.
![Unbenannt](https://github.com/user-attachments/assets/b46951ed-82f5-47a4-b518-e1d1c14e45d2)
![Unbenannt1](https://github.com/user-attachments/assets/b9bfde76-620e-414a-bad0-d1f2791ce984)

**Reply by SybexX** [reply]
Nachdem ich die conf.ini auf der SD durch die von 15.7.0 ersetzt habe, funktioniert bei mir alles.
